packages/zapp/console/src/components/Launch/LaunchForm/LaunchInterruptibleInput.tsx

```diff
@@ -74,14 +74,14 @@ export const LaunchInterruptibleInputImpl: React.ForwardRefRenderFunction<
   const getInterruptibleLabel = () => {
```

Comment:
Can you please add TODO above this function to cover all text variants in localization conditional?